Sources/Fuzzilli/Fuzzer.swift

```diff
@@ -247,10 +253,15 @@ public class Fuzzer {
     ///
     /// This will import each program in the given array into this fuzzer while potentially discarding
     /// some percentage of the programs if dropout is enabled.
-    public func importCorpus(_ corpus: [Program], withDropout applyDropout: Bool = false) {
+    public func importCorpus(_ corpus: [Program], withDropout applyDropout: Bool = false, logStatus: Bool = false) {
         dispatchPrecondition(condition: .onQueue(queue))
+        var count = 1
         for program in corpus {
-            internalImportProgram(program, withDropout: applyDropout, isCrash: false)
+            internalImportProgram(program, withDropout: applyDropout, isCrash: false, initialCorpusImport: true)
```

Comment:
Yeah, so I think we should select the import mode via another function parameter of this function - either a boolean or an enum. The two possible modes are:
1. Only programs achieving new coverage are imported, and those are also minimized. This will be fairly slow
2. All given samples are added to the corpus regardless of whether they achieve new coverage. Also, they are *not* minimized (IIUC currently, those programs that trigger new coverage are minimized, the others aren't). This mode should be very fast, but it will still need to execute every sample once to ensure that it is valid and to collect coverage and then again to collect runtime type information.

FuzzilliCli/main.swift should then probably select one of these modes depending on commandline arguments

Reply:
Here's an idea. Have one flag w/ file input for where all successfully executed programs end up in the corpus, and a different one where only the ones that increase coverage get added. The master instance would run both, resulting in a corpus that has all prior regressions for that particular engines, and additional programs that happen to hit additional coverage beyond the provided regressions.

Reply:
Makes sense. I'm making `runStartupTests` a public function, and explicitly calling it in from main, to ensure everything is ready to go prior to corpus import.

Reply:
Cool, good idea!